teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(specialization, age, name, parent_permission)
```

Comment:
- [ ] again here the `name` attribute should be set to `"Unknown"` and the `parent_permission` should be set to `true`